src/Documentation/Advanced-Concepts/Dependency-Injection.md

```diff
@@ -109,4 +109,52 @@ public class InjectedService : IInjectedService
         return Task.FromResult(DateTime.UtcNow.Ticks);
     }
 }
-```
\ No newline at end of file
+```
+
+# Test Framework Integration
+
+DI truly shines when coupled with a testing framework to verify the correctness of the code that build. You can read about the components for making testing in Orleans by following our [tutorials](../../Tutorials/Unit-Testing-Grains.md).
```

Comment:
When compiled, does this link turn into a link to the HTML docs or the .md file? (I'm new to this)